File: CS/Q181266/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using DevExpress.XtraEditors.ViewInfo;
using DevExpress.Utils;

namespace Q181266
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        private void checkEdit1_MouseDown(object sender, MouseEventArgs e)
        {
            CheckEdit cEdit = (CheckEdit)sender;
            CheckEditViewInfo cInfo = (CheckEditViewInfo)cEdit.GetViewInfo();
            Rectangle r = cInfo.CheckInfo.GlyphRect;
            Rectangle editorRect = new Rectangle(new Point(0, 0), cEdit.Size);
            if (!r.Contains(e.Location) && editorRect.Contains(e.Location))
                ((DXMouseEventArgs)e).Handled = true;
        }
    }
}

File: VB/Q181266/Form1.vb
Imports System.ComponentModel
Imports System.Drawing
Imports System.Windows.Forms
Imports DevExpress.XtraEditors
Imports DevExpress.XtraEditors.ViewInfo
Imports DevExpress.Utils

Namespace Q181266

    Public Partial Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub checkEdit1_MouseDown(ByVal sender As Object, ByVal e As MouseEventArgs)
            Dim cEdit As CheckEdit = CType(sender, CheckEdit)
            Dim cInfo As CheckEditViewInfo = CType(cEdit.GetViewInfo(), CheckEditViewInfo)
            Dim r As Rectangle = cInfo.CheckInfo.GlyphRect
            Dim editorRect As Rectangle = New Rectangle(New Point(0, 0), cEdit.Size)
            If Not r.Contains(e.Location) AndAlso editorRect.Contains(e.Location) Then CType(e, DXMouseEventArgs).Handled = True
        End Sub
    End Class
End Namespace
